docs/diagrams/CommandInheritance.puml

```diff
@@ -0,0 +1,30 @@
+@startuml
+left to right direction
+abstract Command
+Command <|-- AddCommand
+Command <|-- AddMemberCommand
+Command <|-- AddTransactionCommand
+Command <|-- AddGroupCommand
+Command <|-- DeleteCommand
+Command <|-- DeleteMemberCommand
+Command <|-- DeleteTransactionCommand
+Command <|-- DeleteGroupCommand
```

Comment:
I think you do not need to mention every Command, perhaps mention a few examples. That is enough to give the reader an overview of the inheritance. Currently the diagram looks rather long.